pkg/kwokctl/utils/cmd.go

```diff
@@ -100,7 +100,10 @@ func ForkExecRestart(ctx context.Context, dir string, name string) error {
 
 func ForkExecKill(ctx context.Context, dir string, name string) error {
 	pidPath := PathJoin(dir, "pids", filepath.Base(name)+".pid")
-	if _, err := os.Stat(pidPath); err != nil {
+	_, err := os.Stat(pidPath)
+	if err != nil {
+		fmt.Printf("get fileInfo of pid file %s: %v", pidPath, err)
+		// No pid file exists, which means the process has been terminated
```

Comment:
```suggestion
		// No pid file exists, which means the process has been terminated
		logger := log.FromContext(ctx)
		logger = logger.Debug("Stat file", 
			"path", pidPath,
			"err", err,
		)
```

Reply:
Changed as expected. Thanks for the advice 😁